### PR TITLE
lieutenant envirosuit and skirt and lieutenant+brigphys hud icons too

### PR DESCRIPTION
**Pull request by dinnerwoerror**
## About The Pull Request

exactly what you think it is

i think they look a little nice

Screenshot (955)

**P**

and yes, lieutenant plasma does spawn with combat gloves, because the normal lieutenant gets them

## Why It's Good For The Game

i want to play lieutenant too

## Changelog
:cl:
add: lieutenant envirosuit & skirt
add: lieutenant & brigphys hud icons

### CODE REVIEW

**Comment by triplezeta** [comment]
nice

**P**

**Comment by dinnerwoerror** [comment]
the lieutenant's hud icon might look a little familiar to some

the psychologist's hud icon is also just **P** as well, but with the cyan blue medical department background color

i think there are already a lot of white cross medical staff hud icons, i think the fourth time is the charm here with the medical doctor and paramedic and cmo and medical response officer all having the same white cross but with different background colors

yes, the depiction of the **red** cross icon here is technically a violation of the geneva conventions, but that's beside the point

**Comment by triplezeta** [comment]
difference with the psychologist is that there is already a cyan blue background white cross icon (the medical doctor) while security red background white cross icon is a niche not yet filled

**Review by triplezeta — APPROVED** [review]
works for me

**Review by Superyodeler — CHANGES_REQUESTED** [review]
Really? Again?

File: code/modules/clothing/outfits/plasmaman.dm [hunk 1]
```diff
@@ -211,3 +211,10 @@
 	uniform = /obj/item/clothing/under/plasmaman/clown
 	mask = /obj/item/clothing/mask/gas/clown_hat
 	gloves = /obj/item/clothing/gloves/color/plasmaman/clown
+
+/datum/outfit/plasmaman/lieutenant
+	name = "Lt. Plasma"
+	head = /obj/item/clothing/head/helmet/space/plasmaman/security/lieutenant
+	uniform = /obj/item/clothing/under/plasmaman/lieutenant
+	gloves = /obj/item/clothing/gloves/combat
```
Comment (Superyodeler) [inline code/modules/clothing/outfits/plasmaman.dm:219]: Really?

Comment (triplezeta) [reply code/modules/clothing/outfits/plasmaman.dm:219]: the lieutenant gets combat gloves you moth

Comment (Superyodeler) [reply code/modules/clothing/outfits/plasmaman.dm:219]: Shoot, didn't read

Comment (triplezeta) [reply code/modules/clothing/outfits/plasmaman.dm:219]: image

Comment (Superyodeler) [reply code/modules/clothing/outfits/plasmaman.dm:219]: am dumb

File: code/modules/clothing/under/jobs/Plasmaman/civilian_service.dm [hunk 2]
```diff
@@ -151,3 +151,16 @@
...
 	icon_state = "mime_enviroskirt"
 	item_state = "mime_enviroskirt"
+
+/obj/item/clothing/under/plasmaman/lieutenant
+	name = "lieutenants envirosuit"
+	desc = "A navy blue envirosuit with chlorophyll green trimmings."
+	icon_state = "lt_envirosuit"
+	item_state = "lt_envirosuit"
+
+/obj/item/clothing/under/plasmaman/lieutenant/skirt
```
Comment (Superyodeler) [inline code/modules/clothing/under/jobs/Plasmaman/civilian_service.dm:161]: Margularize
```suggestion
/obj/item/clothing/under/plasmaman/lieutenant/skirt //WS edit plasmaman customization
```

File: code/modules/clothing/under/jobs/Plasmaman/civilian_service.dm [hunk 3]
```diff
@@ -151,3 +151,16 @@
 	desc = "It's very colourful on the inside."
 	icon_state = "mime_enviroskirt"
 	item_state = "mime_enviroskirt"
+
+/obj/item/clothing/under/plasmaman/lieutenant
```
Comment (Superyodeler) [inline code/modules/clothing/under/jobs/Plasmaman/civilian_service.dm:155]: Margularize
```suggestion
/obj/item/clothing/under/plasmaman/lieutenant //WS edit plasmaman customization
```

File: code/modules/clothing/spacesuits/plasmamen.dm [hunk 4]
```diff
@@ -283,39 +283,46 @@
...
 	item_state = "hop_envirohelm"
-	
+
 /obj/item/clothing/head/helmet/space/plasmaman/security/secmed
 	name = "brig physician envirosuit helmet"
 	desc = "An envirosuit helmet made for the sanctioned plasmaman."
 	icon_state = "secmed_envirohelm"
 	item_state = "secmed_envirohelm"
+
+/obj/item/clothing/head/helmet/space/plasmaman/security/lieutenant
```
Comment (Superyodeler) [inline code/modules/clothing/spacesuits/plasmamen.dm:323]: Margularize
```suggestion
/obj/item/clothing/head/helmet/space/plasmaman/security/lieutenant //WS edit plasmaman customization
```

File: code/modules/clothing/outfits/plasmaman.dm [hunk 5]
```diff
@@ -211,3 +211,10 @@
 	uniform = /obj/item/clothing/under/plasmaman/clown
 	mask = /obj/item/clothing/mask/gas/clown_hat
 	gloves = /obj/item/clothing/gloves/color/plasmaman/clown
+
+/datum/outfit/plasmaman/lieutenant
```
Comment (Superyodeler) [inline code/modules/clothing/outfits/plasmaman.dm:215]: Margularize
```suggestion
/datum/outfit/plasmaman/lieutenant //WS edit plasmaman customization
```

**Review by Superyodeler — CHANGES_REQUESTED** [review]
Actual, semi-intelligent change request.

File: code/modules/clothing/outfits/plasmaman.dm (hunk 1 above)
Comment (Superyodeler) [reply code/modules/clothing/outfits/plasmaman.dm:219]: Am dumb

Comment (dinnerwoerror) [reply code/modules/clothing/outfits/plasmaman.dm:219]: right there, in case you missed it

Screenshot (956)

**Review by Superyodeler — CHANGES_REQUESTED** [review]
An entry also needs to be added to this switch statement for the outfit to appear ingame: https://github.com/WaspStation/WaspStation-1.0/blob/d8b3dcb31b4a8f9b06bb7092337a8e499e5ad71d/code/modules/mob/living/carbon/human/species_types/plasmamen.dm#L72

File: code/modules/clothing/outfits/plasmaman.dm (hunk 1 above)
Comment (Superyodeler) [reply code/modules/clothing/outfits/plasmaman.dm:219]: :moffdonk:

**Comment by dinnerwoerror** [comment]
there we go

File: code/modules/mob/living/carbon/human/species_types/plasmamen.dm [hunk 6]
```diff
@@ -159,6 +159,9 @@
 
 		if("Clown")
 			O = new /datum/outfit/plasmaman/clown
+		
+		if("Lieutenant")
+			O = new /datum/outfit/plasmaman/lieutenant
```
Comment (Superyodeler) [inline code/modules/mob/living/carbon/human/species_types/plasmamen.dm:164]: arg
```suggestion
		if("Lieutenant")	//WS edit plasmaman customization
			O = new /datum/outfit/plasmaman/lieutenant	//WS edit plasmaman customization
```

**Review by Superyodeler — CHANGES_REQUESTED** [review]
Margularize

File: code/modules/clothing/outfits/plasmaman.dm (hunk 1 above)
Comment (dinnerwoerror) [reply code/modules/clothing/outfits/plasmaman.dm:219]: it still boggles me how the lieutenant has to be the odd one out

**Review by Superyodeler — APPROVED** [review]
Works fine

**Pull request merged**